### PR TITLE
missing 'sync-client' and 'sqlit-jdbc' dependencies

### PR DESCRIPTION
**Pull request by thesherrinford**
As suggested by @linl33 , I've installed the jars in 'dependencies' (in a way, moved jars to 'dependencies' from 'target')

Thank you!

AdityA :)

### CODE REVIEW

**Comment by wbrunette** [comment]
@thesherrinford. Version 2.1.2 was released yesterday so it would preferable to update to the latest jars.

File: build.gradle [hunk 1]
```diff
@@ -13,7 +13,7 @@ repositories {
 
 dependencies {
 
-    runtime files ('target/sync-client-2.1.1.jar','target/sync-client-2.1.1-javadoc.jar','target/sync-client-2.1.1-sources.jar')
+    runtime files ('dependencies/sync-client-2.1.1.jar','dependencies/sync-client-2.1.1-javadoc.jar','dependencies/sync-client-2.1.1-sources.jar')
```
Comment (linl33) [inline build.gradle]: It doesn't compile with `runtime`
```suggestion
    implementation files ('dependencies/sync-client-2.1.1.jar','dependencies/sync-client-2.1.1-javadoc.jar','dependencies/sync-client-2.1.1-sources.jar')
```

File: build.gradle [hunk 2]
```diff
@@ -13,7 +13,7 @@ repositories {
...
     runtime fileTree('target') {include'*.jar'}
```
Comment (linl33) [inline build.gradle]: This was added by your previous pull request and we don't need this anymore. 
```suggestion
```

**Comment by thesherrinford** [comment]
Ok sir.
Thank you!
I'll do the changes

**Comment by thesherrinford** [comment]
@wbrunette yes sir. Thank you!
I'll update it

**Pull request closed without merging**